lib/models/notices.js

```diff
@@ -249,7 +249,7 @@ function processCBv2Notice(service, subservice, ncr, ix) {
         var attrList = ncr.data[ix];
         for (var attr in attrList) {
             // Exclude id and type. NGSIv2
-            if (attr === 'id' || attr === 'type') {
+            if (attr !== 'id' && attr !== 'type') {
```

Comment:
Expression `(attr === 'id' || attr === 'type')` is not equivlante to `(attr !== 'id' && attr !== 'type')`.

Or instance if attr is equal to "id" then:

```
(attr === 'id' || attr === 'type') =>  (True || False) => True
```

while

```
(attr !== 'id' && attr !== 'type') => (False && True) => False
```

Reply:
Fixed in 08201e79ae90920e0f9707b01629f3f281c7d4ad